packages/renderer/src/lib/ui/Button.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+import type { ButtonType } from './Button';
+import { onMount } from 'svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import Spinner from './Spinner.svelte';
+
+export let title: string = undefined;
+export let inProgress = false;
+export let disabled = false;
+export let type: ButtonType = 'primary';
+export let icon: any = undefined;
+
+let iconType: string = undefined;
+
+onMount(() => {
+  if (icon?.prefix === 'fas') {
+    iconType = 'fa';
+  } else if (icon?.name.includes('PodIcon')) {
+    iconType = 'pd';
```

Comment:
I'm wondering if we should not change defaults of PodIcon to be solid/1em size by default
so you wouldn't need to handle specifically PodIcon in buttons

Reply:
I was debating that, but if we do it we should really change default for the matching container/image/volume icon too (although I don't think we don't use them on buttons yet, they are identical pattern and 1em solid seems like a reasonable default..).

Trying to limit the blast radius of this PR. Ok if I took that as separate cleanup-PR?

Reply:
yes sure it's another PR